aries-site/src/themes/aries.js

```diff
@@ -250,15 +397,42 @@ export const aries = {
     },
   },
   text: {
-    small: fontSizing(0), // 18
-    medium: fontSizing(1), // 24
-    large: fontSizing(3), // 36
-  },
-  paragraph: {
-    small: fontSizing(0),
-    medium: fontSizing(1),
-    large: fontSizing(3),
+    xsmall: {
+      size: '12px',
+      height: '18px',
+      maxWidth: '288px',
+    },
+    small: {
+      size: '14px',
+      height: '20px',
+      maxWidth: '336px',
+    },
+    medium: {
+      size: '18px',
```

Comment:
Because 'medium' is the default, I would recommend that it be defined as the 24px size, then small = 18px.

Reply:
I agree, checking what is the right scaling to get the site values back. At the moment I'm thinking it is relying on the factors. WIP